advanced-achievements-plugin/src/main/java/com/hm/achievement/listener/statistics/CraftsListener.java

```diff
@@ -57,8 +57,8 @@ public void onCraftItem(CraftItemEvent event) {
 			return;
 		}
 
-		Set<String> foundAchievements = findAchievementsByCategoryAndName(category, craftName + ":" + item.getDurability());
-		foundAchievements.addAll(findAchievementsByCategoryAndName(category, craftName));
+		Set<String> foundAchievements = findAchievementsByCategoryAndName(category, craftName + ':' + item.getDurability());
+		foundAchievements.addAll(findAchievementsByCategoryAndName(category, craftName + ':' + item.getDurability()));
```

Comment:
I believe this was correct: first retrieve all achievements matching the craft name with the metadata, then add all achievements matching the craft name without the metadata. The plugin supports both with and without metadata for this category. 😉

Reply:
What I found was that if that wasn't included then creating metadata wood would also count towards regular wood. Not sure if that's another problem, but with that part added to it, I was able to craft wood and metadata wood fine without them conflicting.

Reply:
https://github.com/PyvesB/AdvancedAchievements/blob/4cb6d042c81a3ad9b495795d0b136d2765bd1e0d/advanced-achievements-plugin/src/main/java/com/hm/achievement/listener/statistics/AbstractListener.java#L110-L122

One of them should be with durability and one without. Wouldn't this change break achievements with just `wood`.

Reply:
Yea I see now, it would break it for just wood, but without it you can craft wood:1 and get the points towards wood as well as wood:1. I think the best fix would be to see if the durability is above 0 and if it is then add that part.
```java
if(item.getDurability() > 0) {
    foundAchievements.addAll(findAchievementsByCategoryAndName(category, craftName + ':' item.getDurability()));
} else {
    foundAchievements.addAll(findAchievementsByCategoryAndName(category, craftName));
}
```

Reply:
But isn't metadata wood simply a subset of wood? The player has crafted wood, so he should get any related achievements, and in particular he crafted spruce (or whatever) wood so he should get any related achievements as well. Therefore I would be inclined to think that both statistics should increase when crafted. 😉

Reply:
I wouldn't say so, if you have achievements to craft every type of wood, why would you be able to receive multiple by crafting one type of wood. In my opinion all wood types should be considered seperate. Maybe this could be something that could be changed a bit. Such as an additional achievement that could differentiate between all wood types and regular wood.

Reply:
I thought about it, you could do wood:0 for just regular wood, correct? In that case ignore what I said. Might just be good to make users aware of the difference between wood and wood:0.

Reply:
I'm unsure what you mean by "regular" wood, but I believe wood:0 is oak wood.